### ISSUE TITLE
API for captions?

### ISSUE THREAD

**Issue by sjcrs** [beaufour/shutterfly_sites_api#4]
Hi, I'm curious if your API explorations revealed the text from picture captions.  My wife is devastated that we might lose dates and captions for all of our share site pictures.

**Comment by beaufour** [comment]
Yes, both titles and dates are available through the API. Note though that it's not super well tested, as it likely has only been used by me :)
